### PR TITLE
Add landing message. #1 #2

### PR DESCRIPTION
**Pull request by faiz2harry**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,6 @@
   <title>Welcome to Potionfy</title>
 </head>
 <body>
-  
+    <h1>Welcome to Potionfy! Enjoy your stay.</h1>
```
Comment (faiz2hermione) [inline index.html]: Could you remove the 'Enjoy your stay.' string and make the 'Welcome to Potionfy!' string red?

Comment (faiz2harry) [reply index.html]: Sure, can do!

**Review by faiz2hermione — CHANGES_REQUESTED** [review]
Please, change the code as per my comments on the review.

**Review by faiz2hermione — APPROVED** [review]
Well done Harry!

**Pull request merged**